tests/unit/mpool/mblock_test.c

```diff
@@ -347,7 +348,7 @@ mblock_rw(struct mpool *mp, uint64_t mbid, void *buf, size_t len, u64 boff, bool
     iovc = 0;
 
     /* Is the caller's buffer page aligned? */
-    if ((u64)pos & (u64)(PAGE_SIZE - 1)) {
+    if ((uintptr_t)pos & (uint64_t)(PAGE_SIZE - 1)) {
```

Comment:
Drive-by cleanup?

Reply:
Nah, the includes are totally f-ed up, so when event counter no longer got included by logging, this was all the fallout. It took my liked an hour to slog through this. Totally annoying. This is why files should include what they use!

Reply:
Ah, that old irritant.  The sooner we get rid of all our custom integer type names the better IMO.